### PR TITLE
Add `cts_login` and `cts_logout` events

### PR DESCRIPTION
**Pull request by rrrrs09**
# Description

Added `cts_login` and `cts_logout` events.

# Checklist

- [x] Linters and tests have passed
- [ ] I've written a changelog for PR change
- [ ] I'll make a release when PR is approved

### CODE REVIEW

File: botx/models/system_events/cts_login.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    from typing_extensions import Literal  # type: ignore  # noqa: WPS440
+
+
+@dataclass
+class CTSLoginEvent(BotCommandBase):
+    """Event `system:cts_login`.
+
+    Attributes:
+        user_huid: user ID.
+        cts_id: ID of the server the user is logged in to.
```
Comment (lxmnk) [inline botx/models/system_events/cts_login.py:27]: `into`.

Comment (lxmnk) [inline botx/models/system_events/cts_login.py:27]: We can't use this field, maybe just drop it?

File: botx/models/system_events/cts_logout.py [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    from typing_extensions import Literal  # type: ignore  # noqa: WPS440
+
+
+@dataclass
+class CTSLogoutEvent(BotCommandBase):
+    """Event `system:cts_logout`.
+
+    Attributes:
+        user_huid: user ID.
+        cts_id: ID of the server the user is logged out from.
```
Comment (lxmnk) [inline botx/models/system_events/cts_logout.py:27]: We can't use this field, maybe just drop it?

File: botx/models/system_events/cts_logout.py [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+class BotAPICTSLogoutData(VerifiedPayloadBaseModel):
+    user_huid: UUID
+    cts_id: UUID
+
+
+class BotAPICTSLogoutPayload(VerifiedPayloadBaseModel):
+    body: Literal["system:cts_logout"] = "system:cts_logout"
+    command_type: Literal[BotAPICommandTypes.SYSTEM]
+    data: BotAPICTSLogoutData
+    metadata: Dict[str, Any]
```
Comment (lxmnk) [inline botx/models/system_events/cts_logout.py:43]: Drop `metadata` from check, you don't use it.

File: botx/models/system_events/cts_login.py [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+class BotAPICTSLoginData(VerifiedPayloadBaseModel):
+    user_huid: UUID
+    cts_id: UUID
+
+
+class BotAPICTSLoginPayload(VerifiedPayloadBaseModel):
+    body: Literal["system:cts_login"] = "system:cts_login"
+    command_type: Literal[BotAPICommandTypes.SYSTEM]
+    data: BotAPICTSLoginData
+    metadata: Dict[str, Any]
```
Comment (lxmnk) [inline botx/models/system_events/cts_login.py:43]: Drop `metadata` from check, you don't use it.

**Review by Nimond — APPROVED** [review]

File: botx/models/system_events/cts_login.py (hunk 1 above)
Comment (rrrrs09) [reply botx/models/system_events/cts_login.py]: done

File: botx/models/system_events/cts_logout.py (hunk 2 above)
Comment (rrrrs09) [reply botx/models/system_events/cts_logout.py]: done

File: botx/models/system_events/cts_logout.py (hunk 3 above)
Comment (rrrrs09) [reply botx/models/system_events/cts_logout.py]: done

File: botx/models/system_events/cts_login.py (hunk 4 above)
Comment (rrrrs09) [reply botx/models/system_events/cts_login.py]: done

**Review by lxmnk — APPROVED** [review]

**Pull request merged**